spring-integration-file/src/main/java/org/springframework/integration/file/dsl/FileInboundChannelAdapterSpec.java

```diff
@@ -83,14 +100,14 @@ FileInboundChannelAdapterSpec directory(File directory) {
 	 */
 	public FileInboundChannelAdapterSpec scanner(DirectoryScanner scanner) {
 		this.scanner = scanner;
-		this.target.setScanner(scanner);
+
```

Comment:
Why is this removed?

Reply:
How is this resolved? How does an external scanner get applied now?

Reply:
It is restored in my local copy. 
Will push soon enough!
Sorry for confusion 😄